<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="SunilNad_Lenevo330" name="tests.Test1" tests="14" failures="3" timestamp="2021-07-01T20:06:33 IST" time="233.165" errors="0">
  <testcase name="method5" time="11.224" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method2" time="13.869" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method5" time="22.794" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method1" time="16.006" classname="tests.Test1">
    <failure type="java.lang.AssertionError" message="failing... ">
      <![CDATA[java.lang.AssertionError: failing... 
at org.testng.Assert.fail(Assert.java:99)
at tests.Test1.method1(Test1.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- method1 -->
  <system-out/>
  <testcase name="method4" time="13.680" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method5" time="18.706" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method3" time="13.281" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method1" time="8.550" classname="tests.Test1">
    <failure type="java.lang.AssertionError" message="failing... ">
      <![CDATA[java.lang.AssertionError: failing... 
at org.testng.Assert.fail(Assert.java:99)
at tests.Test1.method1(Test1.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- method1 -->
  <system-out/>
  <testcase name="method3" time="23.757" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method1" time="23.729" classname="tests.Test1">
    <failure type="java.lang.AssertionError" message="failing... ">
      <![CDATA[java.lang.AssertionError: failing... 
at org.testng.Assert.fail(Assert.java:99)
at tests.Test1.method1(Test1.java:21)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- method1 -->
  <system-out/>
  <testcase name="method4" time="24.773" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method3" time="10.364" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method4" time="9.445" classname="tests.Test1"/>
  <system-out/>
  <testcase name="method2" time="22.987" classname="tests.Test1"/>
  <system-out/>
</testsuite> <!-- tests.Test1 -->
